### ISSUE TITLE
Swedish grammar - dates

### ISSUE THREAD

**Issue by walseb** [languagetool-org/languagetool#9819]
Hello!

The rules for Swedish grammar appear quite rudimentary.

One simple rule that's missing is for week days. Week days should be written with uncapitalized letters.

Source:
http://dinsvenska.se/stor-bokstav/ska-manader-och-veckodagar-inledas-med-stor-eller-liten-bokstav/

Just thought I would add this issue as I don't know the syntax for adding new rules.

**Comment by danielnaber** [comment]
Indeed, nobody is working on LT's support for Swedish currently. If you'd like to contribute, here's a documentation about how to write simple rules: https://dev.languagetool.org/development-overview

**Comment by ljo** [comment]
I have some rules coming for this. I just got stuck with some dictionary extension and remodelling during the autumn. Please assign it to me.
